### ISSUE TITLE
Delay Inline Completion Audio Cues 

### ISSUE THREAD

**Issue by marrej** [microsoft/vscode#208449]
TLDR: We got some reports that the inline suggest audio cues mask/annoy users when suggest dropdown is triggered, allowing users with screen readers to select when the cue should be played would be helpful.

## Current state
Currently the inline completion audio cues are played right when the inline completion appears on screen. If a suggest dropdown triggers the inline completion, the alert cue is played at the time when the suggest dropdown is red. Causing annoyance and possible confusion for the users.

## Proposal
Provide a optional setting for the inline suggest audio cues, that would allow to select when a cue is played.

- When no suggest dropdown is rendered (playing audio cue, only if a suggest dropdown has been resolved and inline completion is shown)
- Every time inline suggest is rendered (the current state)

This would allow the screen reader users to keep the audio cues, but decide whether and when they should be played.

**Comment by hediet** [comment]
I don't think a setting is necessary here - when the suggest widget is visible, inline completion signals just shouldn't be played.
